packages/atlas/src/themesource/atlas_core/native/core/helpers/_functions/device.ts

```diff
@@ -13,6 +13,17 @@ const isiPhoneModelWithNotch = (): boolean => {
             case "iPhone12,1": // Iphone 11
             case "iPhone12,3": // Iphone 11 Pro
             case "iPhone12,5": // Iphone 11 Pro Max
+            case "iPhone13,1": // iPhone 12 mini
+            case "iPhone13,2": // iPhone 12
+            case "iPhone13,3": // iPhone 12 Pro
+            case "iPhone13,4": // iPhone 12 Pro Max
+            case "iPhone14,4": // iPhone 13 mini
+            case "iPhone14,5": // iPhone 13
+            case "iPhone14,2": // iPhone 13 Pro
+            case "iPhone14,3": // iPhone 13 Pro Max
+            case "iPhone14,8": // iPhone 14 Plus
```

Comment:
Missing "iPhone 14" which the id is iPhone14,7

Reply:
Thanks Diego, good catch. I added it.